fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -78,11 +78,31 @@ To gradually roll out a new build use
 fastlane supply --apk path/app.apk --track rollout --rollout 0.5
 ```
 
+### Expansion files (`.obb`)
+
 Expansion files (obbs) found under the same directory as your APK will also be uploaded together with your APK as long as:
 
 - they are identified as type 'main' or 'patch' (by containing 'main' or 'patch' in their file name)
 - you have at most one of each type
 
+If you only want to update the APK, but keep the expansion files from the previous version on Google Play use
+
+```no-highlight
+fastlane supply --apk path/app.apk --obb_main_references_version version --obb_main_file_size size
+```
+
+or
+
+```no-highlight
+fastlane supply --apk path/app.apk --obb_patch_references_version version --obb_patch_file_size size
+```
+
+Example:
```

Comment:
And remove this whole example to keep it shorter then.

Reply:
Example removed, used values instead of variables.